server/conf/application.test.conf

```diff
@@ -17,3 +17,5 @@ azure.blob.container = "super cool blob container name"
 azure.blob.account = "my awesome azure account name"
 
 application_status_tracking_enabled = true
+
+auth.applicant_idp = "disabled"
```

Comment:
Nit: This seems unrelated. Mind splitting it out separately or explicitly updating the PR description to call out this change?

Reply:
It's not related, but it is needed to get the tests to work.  I split it out into https://github.com/seattle-uat/civiform/pull/2927. (I assume this is also what is causing other tests to fail)